tests/Aspire.Hosting.PostgreSQL.Tests/Aspire.Hosting.PostgreSQL.Tests.csproj

```diff
@@ -14,6 +14,7 @@
   <ItemGroup>
     <Compile Include="$(SharedDir)SecretsStore.cs" Link="Utils\SecretsStore.cs" />
     <Compile Include="$(RepoRoot)src\Aspire.Hosting\ApplicationModel\UserSecretsParameterDefault.cs" Link="Utils\UserSecretsParameterDefault.cs" />
+    <Compile Include="$(RepoRoot)src\Aspire.Hosting.Testing\ResourceLoggerForwarderService.cs" Link="Utils\ResourceLoggerForwarderService.cs" />
```

Comment:
Instead of this, can we reference Aspire.Hosting.Testing directly? That way we don't need to compile the source into the test. We can use the product.

Reply:
It's internal right now, wasn't sure if we wanted to make it public (I assumed this would have been public in the first place).

Reply:
Made it public